src/evaluate.py

```diff
@@ -79,6 +88,11 @@ def main():
     # # Log the results to wandb as metrics
     config_dict = {k: v for k, v in results['config'].items() if isinstance(v, (str, int))}
     config_dict['num_fewshot'] = config.eval.num_fewshot
+    
```

Comment:
I think you need to remove lines 92-95 because the logging is done in lines 100 upwards